packages/static-pods/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "static-pods"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[package.metadata.build-package]
+source-groups = [
+    "api/static-pods"
+]
```

Comment:
If `static-pods` no longer depends on the API, it should move out from under `sources/api` so that this package can watch the specific directory (`sources/static-pods`) and so that changes don't also trigger a rebuild of `os`.